app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -36,6 +36,7 @@ def update
         if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
```

Comment:
(I’m responding to @n1zyy's comment here so the thread can later be resolved.  We also chatted in slack.)

Based on conversations with the dev who wrote the ticket, I don’t think it’s a bug for a user to be able to re-enter remote idv after starting IPP.  The intention of the ticket was to enable users to switch back to remote idv after starting IPP.  If there is a concern about which specific page users are taken to when they hit the back button and re-enter remote idv, I will say I think that concern is out of scope of LG-15739.

Reply:
I definitely agree that it's out of scope. It also seems like I misunderstood the intent: I thought an IPP user was _unexpectedly_ getting into doc auth.

Reply:
> I thought an IPP user was unexpectedly getting into doc auth, not purposefully switching back.

Ah!  Yeah, this scenario is when the user **intentionally** re-enters remote idv.